src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import core.basesyntax.converter.DataConverter;
+import core.basesyntax.converter.DataConverterImpl;
+import core.basesyntax.dao.FileReader;
+import core.basesyntax.dao.FileReaderImpl;
+import core.basesyntax.generator.ReportGenerator;
+import core.basesyntax.generator.ReportGeneratorImpl;
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.handler.OperationStrategy;
+import core.basesyntax.handler.OperationStrategyImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operations.BalanceOperation;
+import core.basesyntax.operations.PurchaseOperation;
+import core.basesyntax.operations.ReturnOperation;
+import core.basesyntax.operations.SupplyOperation;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.writer.FileWriterService;
+import core.basesyntax.writer.FileWriterServiceImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] arg) {
```

Comment:
The method parameter should be `args` instead of `arg` to match the standard Java `main` method signature.